### ISSUE TITLE
ear trumpet volume control off screen

### ISSUE THREAD

**Issue by p5hema2** [File-New-Project/EarTrumpet#1204]
### Summary

On a quad monitor setup the volume control is partly off screen and therefore not fully usable.

### Steps to reproduce

Use multiple Monitors.
The external Monitors are 4k.
The external Monitors have scaling enabled to 150%.

(Found no way to correct reproduce the error, it appears on all setting combinations. e.g.
Monitor 1,2,3 or 4 set to primary
addjusting scale
addjusting resolution
...)

The bug appears on all monitors.

### EarTrumpet version

2.2.0.0

### Windows version

10.0.19044.1826

### Additional information

![image](https://user-images.githubusercontent.com/1798189/184497464-fbffa684-d935-4a19-9373-47c3fd2286f8.png)

**Comment by riverar** [comment]
@p5hema2 Is this a laptop that is being docked/undocked? Are you using Start11?

**Comment by p5hema2** [comment]
@riverar My laptop is docket.

Left --> Docking Station
Middle --> HDMI Laptop
Right --> Docking Station
Far Right --> Laptop Screen

Start11 --> No, doesn't even know what Start11 is.

**Comment by riverar** [comment]
@p5hema2 Does restarting EarTrumpet fix the issue?

**Comment by p5hema2** [comment]
@riverar yes

**Comment by TDLCTV** [comment]
Easy to reproduce but i think it's a bug from windows in general:
- Start windows on any normal screen without scaling (mine is set to 1440p scale set at 100%)
- Switch to any screen with scale different from the default screen scaling and make it primary screen (My TV is set to 4k scaled at 200%) 
- Now ear trumpet is off screen and unusable until i restart the app
![image](https://user-images.githubusercontent.com/10427316/191411803-c333fece-53eb-4b2a-aa2d-5467e1000627.png)

**Comment by Florre4** [comment]
I have found the same issue, not sure if it's a bug in windows with the values that it gives for the screen resolution when calling the api or in EarTrumpet code. 

My setup is a 4K monitor (left) and a 1440p monitor (right).
When I move the main taskbar to another monitor and then click eartrumpet for the First time, it either shows partly offscreen on the 4K monitor, or with an offset on the 1440p one. 

Clicking it the second time fixes the problem. 

I decided pull the code and debug the main branch and did find some values would be different when the error occured, in FlyoutWindow.xaml.cs.PositionWindowRelativeToTaskbar(WindowsTaskbar.State taskbar) method. (see attached screenshots) 

![EarTrumpetBug](https://user-images.githubusercontent.com/45642489/201351735-090d3082-7a98-4778-a3a2-0d5bf0ebe648.png)

![EarTrumpetBug1](https://user-images.githubusercontent.com/45642489/201352016-2488bb36-2c3a-41d2-aae9-774fbf321f28.jpg)

![EarTrumpetBug2](https://user-images.githubusercontent.com/45642489/201352029-4cb02f4f-7688-4707-a1ee-55ee39c79aa0.jpg)
